release-notes.md

```diff
@@ -24,6 +24,21 @@ Please note this file contains the token @FIREBASE_PROJECT_ID@, this should be r
 
     php /opt/kaltura/app/deployment/updates/scripts/2025_03_12_deploy_firebase_oauth2_notifications.php
 
+## Add channel member subscribed email notification template  ##
+* Issue Type: Task
+* Issue ID: SUP-37427
+
+#### Configuration ####
+None.
+
+### Deployment scripts ###
+First replace all tokens in the XML file below and remove ".template" from the file name, then run the php deployment script.
+
+Add script:
+     - deployment/updates/scripts/xml/2025_03_17_addChannelMemberSubscribedEmailNotification.xml
```

Comment:
I'd put the original file name *.template.xml
and move it below the line above
_First replace all tokens in the XML file below ...._
_.     deployment/updates/scripts/xml/2025_03_17_addChannelMemberSubscribedEmailNotification.template.xml_